### ISSUE TITLE
API behavior does not match documentation: priorities ordering

### ISSUE THREAD

**Issue by dermotduffy** [hyperion-project/hyperion.ng#964]
- [x] I confirm that this is an issue rather than a question.

## Bug report

The 'serverinfo' API command is supposed to return the visible priority as the first entry in the priorities list, as per the [documentation](https://docs.hyperion-project.org/en/json/ServerInfo.html#priorities):

```
visible: If "true" this source is displayed and will be pushed to the led device. The visible:true-source is always the first entry!
```

Example call to the API (cutting out the priorities):

 ```
    "priorities": [
      {
        "active": false,
        "componentId": "EFFECT",
        "origin": "System",
        "owner": "Rainbow swirl fast",
        "priority": 0,
        "visible": false
      },
      {
        "active": true,
        "componentId": "V4L",
        "origin": "System",
        "owner": "V4L2:/dev/video0",
        "priority": 240,
        "visible": true
      },
      {
        "active": true,
        "componentId": "EFFECT",
        "origin": "System",
        "owner": "Warm mood blobs",
        "priority": 254,
        "visible": false
      }
    ],
```

#### Steps to reproduce

Send an RPC call to hyperion-ng:

```
echo '{"command":"serverinfo"}' | nc  hyperion 19444 | jq '.' 
```

#### What is expected?

Priorities should be returned with the visible entry first.

#### What is actually happening?

The visible entry is not first.

#### System

```
Hyperion Server: 
- Build:       (HEAD detached at ffa2fd45) (Paulchen-Panther-69e9fa6d/ffa2fd45-1595517126)
- Build time:  Jul 24 2020 15:49:41
- Git Remote:  https://github.com/hyperion-project/hyperion.ng
- Version:     2.0.0-alpha.7
- UI Lang:     auto (BrowserLang: en-US)
- UI Access:   default
- Avail Capt:  dispmanx,v4l2,framebuffer,qt

Hyperion Server OS: 
- Distribution: Raspbian GNU/Linux 10 (buster)
- Arch:         arm
- Kernel:       linux (5.4.51+ (WS: 32))
- Browser:      Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.105 Safari/53
```

**Comment by Paulchen-Panther** [comment]
I can only notice this behavior when manual priority selection is active.

**Comment by Paulchen-Panther** [comment]
@dermotduffy 
Fixed in PR #1036 .

**Comment by dermotduffy** [comment]
@Paulchen-Panther Verified fixed! (At least in basic experimentation)
